pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -299,9 +316,61 @@ func (o *UseWorkspaceOptions) Run(ctx context.Context) error {
 		return err
 	}
 
+	bindings, err := o.getAPIBindings(ctx, o.kcpClusterClient, newServerHost)
+	if err != nil {
+		// display the error, but don't stop the current workspace from being reported.
+		fmt.Fprintf(o.ErrOut, "error checking APIBindings: %v", err)
+	}
+	if err := findUnresolvedPermissionClaims(o.Out, bindings); err != nil {
+		// display the error, but don't stop the current workspace from being reported.
+		fmt.Fprintf(o.ErrOut, "error checking APIBindings: %v", err)
+	}
+
 	return currentWorkspace(o.Out, newServerHost, shortWorkspaceOutput(o.ShortWorkspaceOutput), workspaceType)
 }
 
+// getAPIBindings retrieves APIBindings within the workspace
+func getAPIBindings(ctx context.Context, kcpClusterClient kcpclient.ClusterInterface, host string) ([]apisv1alpha1.APIBinding, error) {
+	_, clusterName, err := pluginhelpers.ParseClusterURL(host)
+	if err != nil {
+		return nil, err
+	}
+
+	apiBindings, err := kcpClusterClient.Cluster(clusterName).ApisV1alpha1().APIBindings().List(ctx, metav1.ListOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, err
+	} else if apierrors.IsNotFound(err) {
```

Comment:
You'll only get a not-found on a list request if the API does not exist. If there are no APIBindings, you'll get a nil error along with an empty list.

I think you can simplify this to return `nil, err` if err is not nil

Reply:
Ok, that makes sense. So `List` calls return `nil` instead of an HTTP 4xx when there are no items?